pooch/tests/test_core.py

```diff
@@ -309,3 +309,19 @@ def test_downloader_progressbar(capsys):
         assert printed[:25] == progress
         # Check that the downloaded file has the right content
         check_large_data(fname)
+
+
+def test_ftp_downloader():
+    "Test pooch with ftp downloader"
+    with TemporaryDirectory() as local_store:
+        doggo = create(
+            path=local_store,
+            base_url="ftp://speedtest.tele2.net/",
+            version_dev="master",
+            registry={
+                "10MB.zip": "e5b844cc57f57094ea4585e235f36c78c1cd222262bb89d53c94dcb4d6b3e55d"
+            },
+        )
+        assert doggo.is_available("10MB.zip")
+        fname = doggo.fetch("10MB.zip")
+        assert Path(fname).exists()
```

Comment:
This test is checking a bit too many things at once:

* `FTPDownloader` 
* The protocol inference in `Pooch.fetch` 
* `Pooch.is_available` 

It would be better to have separate checks for `is_available` and `FTPDownloader` as well. This one then serves as an integration test to make sure everything works together. For example, `HTTPDownloader` has standalone tests in its docstring and the integration test with `Pooch.fetch` above. 

It also might be best to use a smaller file to minimize the testing time (maybe 100Kb?).